src/sentry/apidocs/examples/dashboard_examples.py

```diff
@@ -0,0 +1,89 @@
+from drf_spectacular.utils import OpenApiExample
+
+DASHBOARD_OBJECT = {
+    "id": "1",
+    "title": "Dashboard",
+    "dateCreated": "2024-06-20T14:38:03.498574Z",
+    "createdBy": {
```

Comment:
What I understand is that anyone can fetch the dashboard, and that would give them all the details of `Admin`. Is that right? Should we revise this permission? Maybe we shouldn't include a user object but maybe just email.